content/en/logs/guide/docker-logs-collection-troubleshooting-guide.md

```diff
@@ -164,6 +164,17 @@ Docker's default is the json-file logging driver so the Container Agent tries to
 
 4. Mount the YAML file into your container following the instructions in the [Docker Agent documentation][3]. For more information on setting log drivers for Docker containers, [see this documentation][4].
 
+## Logs are missing but only for some containers
+
+The Docker daemon can have performances issues while it is trying to retrieve logs for containers for which it has already stored large logs files on disk. This could lead to read timeouts when the Logs Agent is gathering the containers' logs from the Docker daemon. The Logs Agent outputs a log containing `Restarting reader after a read timeout` when it occurs.
```

Comment:
```suggestion
The Docker daemon can have performances issues while it is trying to retrieve logs for containers for which it has already stored large logs files on disk. This could lead to read timeouts when the Datadog Agent is gathering the containers' logs from the Docker daemon. 
The Datadog Agent outputs a log containing `Restarting reader after a read timeout` when it occurs.
```